medusa/__main__.py

```diff
@@ -2070,6 +2078,51 @@ def shutdown(self, event):
             logging.shutdown()
             os._exit(0)  # TODO: Remove in another PR. There's no need for this one.
 
+    @staticmethod
+    def configure_unrar():
```

Comment:
This is proper review. I'm not going to do the work for you. If you are not ready to improve your code, I don't see how you expect it to be merged.